internal/app/tfsec/cmd/flags.go

```diff
@@ -62,6 +63,7 @@ func init() {
 	rootCmd.Flags().BoolVar(&excludeDownloaded, "exclude-downloaded-modules", excludeDownloaded, "Remove results for downloaded modules in .terraform folder")
 	rootCmd.Flags().BoolVar(&includePassed, "include-passed", includePassed, "Include passed checks in the result output")
 	rootCmd.Flags().BoolVar(&includeIgnored, "include-ignored", includeIgnored, "Include ignored checks in the result output")
+	rootCmd.Flags().BoolVar(&disableIgnores, "no-ignores", disableIgnores, "Do not apply any ignore rules - normally ignored checks will fail")
```

Comment:
I had to read this 3 times
```suggestion
	rootCmd.Flags().BoolVar(&disableIgnores, "no-ignores", disableIgnores, "Ignore statements will have no effect; all Terraform will be included in the checks")
```